tests/sentry/replays/test_organization_replay_count.py

```diff
@@ -103,9 +110,7 @@ def test_simple(self):
         assert response.status_code == 200, response.content
         assert response.data == expected
 
-    def test_simple_return_ids(self):
-        event_id_a = "a" * 32
-        event_id_b = "b" * 32
+    def test_simple_performance(self):
```

Comment:
i'm not sure if the diff is just confusing but could we make sure `test_simple_return_ids` remains the same, and `test_simple_performance` test is added in addition?